installer/mac/postinstall.sh

```diff
@@ -16,5 +16,33 @@ sed -i "" -E "s#\"url\" ?: ?\".*\"#\"url\"\: \"$pod_url\"#g" $newPath
 sed -i "" -E "s#\"minimizeOnClose\" ?: ?([Tt][Rr][Uu][Ee]|[Ff][Aa][Ll][Ss][Ee])#\"minimizeOnClose\":\ $minimize_on_close#g" $newPath
 sed -i "" -E "s#\"launchOnStartup\" ?: ?([Tt][Rr][Uu][Ee]|[Ff][Aa][Ll][Ss][Ee])#\"launchOnStartup\":\ $launch_on_startup#g" $newPath
 
+## Add app to login items
+if [ $launch_on_startup == true ]; then
```

Comment:
looks like in this PR you are also adding auto-launch setup?  and also fixing pod url issue?  generally we should put separate issues in different PRs.